output-comparator/src/main/java/org/mobilitydata/gtfsvalidator/outputcomparator/cli/Arguments.java

```diff
@@ -54,16 +54,22 @@
 
   @Parameter(
       names = {"-o", "--output_base"},
-      description = "Base directory to store the outputs",
+      description = "Base directory to store the outputs.",
       required = true)
   private String outputBase;
 
   @Parameter(
       names = {"-s", "--source_urls"},
-      description = "Path to source urls",
+      description = "Path to source urls.",
       required = true)
   private String sourceUrlPath;
 
+  @Parameter(
+      names = {"-c", "--percent_corrupted_sources"},
+      description = "Maximum percentage of corrupted sources.",
```

Comment:
> Could you also provide an example value? For instance, "Passing a '2' value would mean that no more than 2% of source could be corrupted". 

👍🏾

> Do you need this threshold to be double?

Let's make it a `float` in case usage of decimal is needed.

> Also, could you please add the same for "--new_error_threshold" description?

👍🏾